README.md

```diff
@@ -54,6 +54,22 @@ a different API server, copy `app/config.js` to
 not tracked and will be used instead if it exists.
 
 #### Before opening a pull request
+
+Please configure your editor to using the
+following settings to avoid common code inconsistencies and dirty
+diffs:
+
+* Use soft-tabs set to two spaces.
+* Trim trailing white space on save.
```

Comment:
> But it will also prevent new noisy diffs going forward. :)

Devil's advocate, but I've seen very few times where anyone has added trailing whitespace to a line they haven't changed. So it hasn't created noise in the diffs.

Trimming the whitespace always on save can cause whitespace merge conflicts, however, when there are no otherwise conflicting changes.